composables/useDocumentComments.js

```diff
@@ -23,15 +23,25 @@ export function useDocumentComments(document) {
   /** Stores promises of ongoing comment page fetch requests. */
   const pagesPromises = reactive({})
 
+  const documentValue = computed(() =>
+    documentRef.value && toValue(documentRef) !== null ? toValue(documentRef) : null
+  )
```

Comment:
I'm not sure to understand this, why not simply do:

```suggestion
  const documentValue = computed(() =>  toValue(documentRef) ?? null)
```